src/snowflake/cli/api/project/schemas/native_app/native_app.py

```diff
@@ -24,6 +24,10 @@ class NativeApp(UpdatableModel):
         title="Folder at the root of your project where the build step copies the artifacts.",
         default="output/deploy/",
     )
+    generated_root: Optional[str] = Field(
+        title="Folder at the root of your deploy_root where the build step checks in the generated artifacts. This is also the name of the SQL file that will be created.",
```

Comment:
How about simply:
```
Subdirectory of the deploy root where files generated by the Snowflake CLI will be written.
```

We shouldn't make this overly specific to the main file. I think we can play ostrich with that one for now.